argoverse/evaluation/stereo/eval.py

```diff
@@ -0,0 +1,348 @@
+
+        assert len(pred_fpaths) == len(gt_fpaths)
+
+        if self.num_procs == 1:
+            errors = []
+            for (pred_fpath, gt_fpath, gt_obj_fpath) in zip(pred_fpaths, gt_fpaths, gt_obj_fpaths):
+                error = compute_disparity_error(
+                    pred_fpath,
+                    gt_fpath,
+                    gt_obj_fpath,
+                    self.abs_error_thresholds,
+                    self.rel_error_thresholds,
+                    self.figs_fpath,
+                    save_disparity_error_image=self.save_disparity_error_image,
+                )
+                errors.append(error)
+        else:
+            args = [
+                (
+                    pred_fpath,
+                    gt_fpath,
+                    gt_obj_fpath,
+                    self.abs_error_thresholds,
+                    self.rel_error_thresholds,
+                    self.figs_fpath,
+                    self.save_disparity_error_image,
+                )
+                for (pred_fpath, gt_fpath, gt_obj_fpath) in zip(pred_fpaths, gt_fpaths, gt_obj_fpaths)
+            ]
+            with Pool(self.num_procs) as p:
+                errors = p.starmap(compute_disparity_error, args)
+
+        data = pd.concat(errors)
+        data_sum = data.sum()
+        summary: dict() = dict()
+
+        for abs_error_threshold in self.abs_error_thresholds:
+            d1_all = (
+                data_sum[f"num_errors_bg:{str(abs_error_threshold)}"]
+                + data_sum[f"num_errors_fg:{str(abs_error_threshold)}"]
+            ) / (data_sum["num_pixels_bg"] + data_sum["num_pixels_fg"])
+            d1_fg = data_sum[f"num_errors_fg:{str(abs_error_threshold)}"] / data_sum["num_pixels_fg"]
+            d1_bg = data_sum[f"num_errors_bg:{str(abs_error_threshold)}"] / data_sum["num_pixels_bg"]
+            d1_all_est = (
+                data_sum[f"num_errors_bg_est:{str(abs_error_threshold)}"]
+                + data_sum[f"num_errors_fg_est:{str(abs_error_threshold)}"]
+            ) / (data_sum["num_pixels_bg_est"] + data_sum["num_pixels_fg_est"])
+            d1_fg_est = data_sum[f"num_errors_fg_est:{str(abs_error_threshold)}"] / data_sum["num_pixels_fg_est"]
+            d1_bg_est = data_sum[f"num_errors_bg_est:{str(abs_error_threshold)}"] / data_sum["num_pixels_bg_est"]
+
+            summary[f"all:{str(abs_error_threshold)}"] = d1_all * 100
+            summary[f"fg:{str(abs_error_threshold)}"] = d1_fg * 100
+            summary[f"bg:{str(abs_error_threshold)}"] = d1_bg * 100
+            summary[f"all*:{str(abs_error_threshold)}"] = d1_all_est * 100
+            summary[f"fg*:{str(abs_error_threshold)}"] = d1_fg_est * 100
+            summary[f"bg*:{str(abs_error_threshold)}"] = d1_bg_est * 100
+
+        return summary
+
+
+def compute_disparity_error(
+    pred_fpath: Path,
+    gt_fpath: Path,
+    gt_obj_fpath: Path,
+    abs_error_thresholds: List = [10, 5, 3],
```

Comment:
maybe `[10, 5, 3],` should be named defaults at the top of the file?
```python
# absolute error thresholds in pixels, also used for KITTI stereo eval
DEFAULT_ABS_ERROR_THRESHOLDS = [10,5,3]
```

Reply:
Changed.